test/groovy/templates/PiperPipelineStageInitTest.groovy

```diff
@@ -228,4 +229,49 @@ class PiperPipelineStageInitTest extends BasePiperTest {
         assertThat(stepParams.setupCommonPipelineEnvironment?.customDefaultsFromFiles, is(['my-custom-default-file.yml']))
     }
 
+    @Test
+    void "Parameter skipCheckout skips the checkout call"() {
+        jsr.step.piperPipelineStageInit(
+            script: nullScript,
+            juStabUtils: utils,
+            buildTool: 'maven',
+            stashSettings: 'com.sap.piper/pipeline/stashSettings.yml',
+            skipCheckout: true,
+            scmInfo: ["dummyScmKey":"dummyScmKey"]
+        )
+
+        assertThat(stepsCalled, hasItems('setupCommonPipelineEnvironment', 'piperInitRunStageConfiguration', 'artifactPrepareVersion', 'pipelineStashFilesBeforeBuild'))
+        assertThat(stepsCalled, not(hasItems('slackSendNotification', 'checkout')))
```

Comment:
[minor] why do we have `slackSendNotification?

Reply:
removed